osc/store.py

```diff
@@ -27,6 +27,10 @@ def get_store(path, check=True, print_warnings=False):
         store = git_scm.GitStore(path, check)
         if print_warnings:
             git_scm.warn_experimental()
+    elif os.path.exists(os.path.join(path, "../.git")):
+        store = git_scm.GitStore(os.path.join(path, "../"), check)
+        if print_warnings:
+            git_scm.warn_experimental()
```

Comment:
Again, I'm not convinced that doing this purely based on existence of `../.git` is the right thing to do.

Reply:
done via git cli lockup now.